Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift

```diff
@@ -26,7 +79,12 @@ class MSDistributeViewController: UITableViewController, AppCenterProtocol {
     appCenter.setDistributeEnabled(sender.isOn)
     sender.isOn = appCenter.isDistributeEnabled()
   }
-  
+
+  @IBAction func preStartSwitchUpdated(_ sender: Any) {
+    let startTrackValue = preStartSwitch.isOn ? UpdateTrack.getSelf(by: MSDistribute.updateTrack).state.rawValue : nil
```

Comment:
there is a feedback on other platforms to no use a check box

Reply:
OK let's try to make it consistent between platform if we can. I think this was the original comment: https://github.com/microsoft/appcenter-sdk-android/pull/1354#discussion_r370899599